src/reanimated2/core.ts

```diff
@@ -157,15 +144,12 @@ if (nativeShouldBeMock()) {
 export function getTimestamp(): number {
   'worklet';
   if (Platform.OS === 'web') {
-    return (NativeReanimatedModule as JSReanimated).getTimestamp();
+    return NativeReanimatedModule.getTimestamp();
   }
   return _getTimestamp();
 }
 
-function valueSetter<T extends WorkletValue>(
-  sv: WorkletValueSetterContext,
-  value: T
-): void {
+function valueSetter(sv: any, value: any): void {
```

Comment:
And pass in a generic to the method?
```suggestion
function valueSetter<T>(sv: SharedValue<T>, value: any): void {
```